addons/hr_employee_hourly_wage/security/hr_employee_hourly_wage_security.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="group_employee_hourly_wage_manager" model="res.groups">
+            <field name="name">Administrator</field>
+            <field name="category_id" ref="base.module_category_services_employee_hourly_wage"/>
```

Comment:
No implied_ids? E.g. payroll officer

Reply:
The logic I used is that this permission is implied by the appropriate roles in mrp and timesheet, and that those roles already have implied_ids defined. Is that a valid reasoning?

Reply:
Hmm, I could indeed foresee issues when people start manually granting this permission though